apps.awesim.org/apps/myjobs/templates/Basic_Schrodinger_Serial_Job_Owens/pbs_script.sh

```diff
@@ -0,0 +1,33 @@
+#PBS -N ondemand/sys/myjobs/basic_schrodinger_serial
+#PBS -j oe
+#PBS -l walltime=0:10:00
+#PBS -l nodes=1:ppn=28
+#PBS -l software=glide+1
+#PBS -S /bin/sh
+qstat -f $PBS_JOBID
+export
```

Comment:
Are you using `export` here to get a dump of the environment? If so the cannonical way to do that would be to call `env`. Also, if printing the environment is something that you think will be useful why not wait until after you are done with modifying the environment using `lmod`?